### PR TITLE
docs: patching the callouts in "Applying the GitHub OAuth App Secret"

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Patch the callouts in "Applying the GitHub OAuth App Secret" to clarify usage of the lines.

## What issues does this pull request fix or reference?
[RHDEVDOCS-4443](https://issues.redhat.com/browse/RHDEVDOCS-4443)

## Specify the version of the product this pull request applies to
Merge to `main`

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 1]
```diff
@@ -43,8 +43,8 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+<2> This line depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]:
```suggestion
<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
```

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 2]
```diff
@@ -43,8 +43,8 @@ data:
...
+<3> This line is only added for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc:47]:
```suggestion
<3> This is only added for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
```

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 3]
```diff
@@ -43,8 +43,8 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
+<3> This is only added for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]:
```suggestion
<3> This line is only added for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
```

**Review by amisevsk — APPROVED** [review]

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 4]
```diff
@@ -43,8 +43,8 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
```
Comment (vinokurig) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc:46]:
```suggestion
<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Cloud (github.com), omit this line or enter the default `https://github.com`.
```

Comment (vinokurig) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]:
```suggestion
<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using the GitHub Cloud (github.com), omit this line or enter the default `https://github.com`.
```

Comment (max-cx) [reply modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]: @vinokurig, I could only find two major products on GitHub's website: _GitHub Enterprise Server_ and _GitHub Enterprise Cloud_. So I wonder if rewriting it as follows will achieve what you suggested:
```suggestion
<2> This depends on the GitHub product your organization is using: When hosting repositories on github.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
```

**Review by vinokurig — APPROVED** [review]

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 5]
```diff
@@ -43,8 +43,8 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+<2> This depends on the GitHub product your organization is using: When hosting repositories on github.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]:
```suggestion
pass:[<!-- vale CheDocs.CommonTerms = NO -->]
<2> This depends on the GitHub product your organization is using: When hosting repositories on github.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
pass:[<!-- vale CheDocs.CommonTerms = YES -->]
```

File: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc [hunk 6]
```diff
@@ -43,8 +43,10 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+pass:[<!-- vale CheDocs.CommonTerms = NO -->]
+<2> This depends on the GitHub product your organization is using: When hosting repositories on github.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
+pass:[<!-- vale CheDocs.CommonTerms = YES -->]
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc]:
```suggestion
<2> This depends on the GitHub product your organization is using: When hosting repositories on GitHub.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
```

**Pull request merged**